package.json

```diff
@@ -27,10 +14,6 @@
     "test": "lerna run test"
   },
   "engines": {
-    "npm": ">= 7.0.0"
-  },
-  "dependencies": {
-    "@xrplf/eslint-config": "file:packages/eslint-config",
-    "@xrplf/prettier-config": "file:packages/prettier-config"
+    "npm": ">= 8.0.0"
```

Comment:
I dont think our apps are ready to update to require 8

Reply:
Also i dont think nom should ever be specified this way.

Reply:
this is just to develop this repo